src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -1042,7 +1042,7 @@ private static object DeepCopierHelper(Type t, object original)
         /// </summary>
         /// <param name="t">The type.</param>
         /// <returns>true if <paramref name="t"/> is serializable, false otherwise.</returns>
-        internal static bool HasSerializer(Type t)
+        public static bool HasSerializer(Type t)
```

Comment:
I thought this could prove useful for debugging. For example, one could determine at run-time if a serializer exists for a given type, in both grain and client contexts. 
Considering that:
* The method is synchronous and has no side-effects;
* `Serialize` is already public and it involves locking (see call to `GetSerializer`), 
I thought the risk of exposing the method to be minimal.

That being said, let me know your decision and I will make the required changes.